frontend/rollup.config.js

```diff
@@ -52,5 +52,8 @@ export default [{
         copy({
             targets: [{ src: '../thumbnail.png', dest: 'dist/listing' }]
         }),
+        copy({
```

Comment:
This copy should be moved up. Right now it is in the `listing` part, and it should be part of the `viewer` section (after line 40).